src/main/java/appeng/me/storage/CellInventory.java

```diff
@@ -442,21 +444,58 @@ protected void loadCellItems()
 
 		final int types = (int) this.getStoredItemTypes();
 
-		for( int x = 0; x < types; x++ )
+		for( int slot = 0; slot < types; slot++ )
 		{
-			final ItemStack t = ItemStack.loadItemStackFromNBT( this.tagCompound.getCompoundTag( itemSlots[x] ) );
-			if( t != null )
+			NBTTagCompound compoundTag = this.tagCompound.getCompoundTag( itemSlots[slot] );
+			int stackSize = this.tagCompound.getInteger( itemSlotCount[slot] );
+			loadCellItem( compoundTag, stackSize );
+		}
+
+		// cellItems.clean();
+	}
+
+	private void loadCellItem( NBTTagCompound compoundTag, int stackSize )
+	{
+
+		// Now load the item stack
+		final ItemStack t;
+		try
+		{
+			t = ItemStack.loadItemStackFromNBT( compoundTag );
+			if( t == null )
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Redundant nullcheck of t, which is known to be non-null in appeng.me.storage.CellInventory.loadCellItem(NBTTagCompound, int) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=findbugs%3ARCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE)